MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2021,22 +2021,30 @@ private int GetMinBitrate(int sourceBitrate, int requestedBitrate)
 
         private static double GetVideoBitrateScaleFactor(string codec)
         {
+            // hevc & vp9 - 40% more efficient than h.264
             if (string.Equals(codec, "h265", StringComparison.OrdinalIgnoreCase)
                 || string.Equals(codec, "hevc", StringComparison.OrdinalIgnoreCase)
-                || string.Equals(codec, "vp9", StringComparison.OrdinalIgnoreCase)
-                || string.Equals(codec, "av1", StringComparison.OrdinalIgnoreCase))
+                || string.Equals(codec, "vp9", StringComparison.OrdinalIgnoreCase))
             {
                 return .6;
             }
 
+            // av1 - 50% more efficient than h.264
+            if (string.Equals(codec, "av1", StringComparison.OrdinalIgnoreCase))
+            {
+                return .5;
+            }
+
             return 1;
         }
 
         private static int ScaleBitrate(int bitrate, string inputVideoCodec, string outputVideoCodec)
         {
             var inputScaleFactor = GetVideoBitrateScaleFactor(inputVideoCodec);
             var outputScaleFactor = GetVideoBitrateScaleFactor(outputVideoCodec);
-            var scaleFactor = outputScaleFactor / inputScaleFactor;
+
+            // Don't scale the real bitrate lower than the requested bitrate
+            var scaleFactor = Math.Min(outputScaleFactor / inputScaleFactor, 1);
```

Comment:
Sorry for bringing up this old PR again, I have a quick question:

> Don't scale the real bitrate lower than the requested bitrate.

This means `scaleFactor >= 1`, correct? Shouldn't it be `Math.Max` to get that?

Reply:
heck.. I got input and output inverted at that time. You are right.

Reply:
If we only ever increase the bitrate via this scaling (and later clamp it to any limit) this should also effectively obsolete an issue I opened https://github.com/jellyfin/jellyfin/issues/8344 🎉

Reply:
> This means `scaleFactor >= 1`, correct? Shouldn't it be `Math.Max` to get that?

I have noticed that the picture quality of "standard"-transcodes from hevc to h264 (for the web player) has degraded massively in version 10.8.10 and I just wanted to file an issue for this. But I think this change might be the reason, as `scaleFactor` is now tied to 1. (I wonder why no one else has complained about the loss of transcoding quality yet...)